Orange/data/variable.py

```diff
@@ -560,6 +560,24 @@ def copy(self, compute_value=None, *, name=None, **kwargs):
         return var
 
 
+class TupleList(tuple):
+    def __add__(self, other):
+        if isinstance(other, list):
+            warnings.warn(
+                "DiscreteVariable.values is a tuple; "
+                "support for adding a list will be dropped in Orange 3.27",
+                DeprecationWarning)
+            other = tuple(other)
+        return super().__add__(other)
+
+    def copy(self):
+        warnings.warn(
+            "DiscreteVariable.values is a tuple;"
+            "method copy  will be dropped in Orange 3.27",
```

Comment:
This is the one I am most proud of catching:
2 spaces after copy

Reply:
Congrats!